modules/primo/src/Primo/Ting/Result.php

```diff
@@ -40,6 +43,13 @@ class Result implements TingSearchResultInterface {
    */
   protected $collections;
 
+  /**
+   * Ting facets mapped from the search result.
+   *
+   * @var \Ting\Search\TingSearchFacet[]
+   */
+  protected $facets;
```

Comment:
Perhaps initialize this value to `NULL` explicitly as this state has special meaning to `getFacets()`.